Ginger/GingerCoreNET/Application Models/Learn/APIModels/ApiActionConversionUtils.cs

```diff
@@ -225,7 +225,7 @@ private ObservableList<EnhancedActInputValue> GetAPIActionParams(ObservableList<
             try
             {
                 string requestBody = null;
-                if (!string.IsNullOrEmpty(actionToConvert.GetInputParamValue(ActWebAPIBase.Fields.TemplateFileNameFileBrowser)))
+                if (actionToConvert.GetInputParamValue(ActWebAPIBase.Fields.TemplateFileNameFileBrowser) == ApplicationAPIUtils.eRequestBodyType.TemplateFile.ToString())
```

Comment:
i'm not getting this change, "TemplateFileNameFileBrowserl" input value supposed to contain the actual request template file Path- why you comparing it to static value like "ApplicationAPIUtils.eRequestBodyType.TemplateFile"?

Reply:
Changed